src/function/scalar/string/like.cpp

```diff
@@ -522,6 +522,11 @@ void LikeFun::RegisterFunction(BuiltinFunctions &set) {
 	                               nullptr, ILikePropagateStats<NotILikeOperatorASCII>));
 }
 
+ScalarFunction LikeFun::GetFunction() {
+	return ScalarFunction("~~", {LogicalType::VARCHAR, LogicalType::VARCHAR}, LogicalType::BOOLEAN,
```

Comment:
Can we call this function in `LikeFun::RegisterFunction` to avoid duplicated logic?